package.json

```diff
@@ -107,8 +101,18 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable/disable Unibeautify formatter"
+                },
+                "unibeautify.defaultConfig": {
+                    "type": "string",
+                    "default": null,
+                    "description": "Type in a path to your .unibeautifyrc outside the workspace folder. This will override the projects .unibeautifyrc config file. Note: The file path must be absolute for a directory or file!"
```

Comment:
Consider rephrasing:

```diff
- "description": "Type in a path to your .unibeautifyrc outside the workspace folder. This will override the projects .unibeautifyrc config file. Note: The file path must be absolute for a directory or file!"
+ "description": "Absolute path to your .unibeautifyrc file outside the workspace folder. This will override the workspace's own .unibeautifyrc config file."
```